user_guide/data_management_data_audit.rst

```diff
@@ -26,8 +36,17 @@ The report grid contains the following columns:
 .. image:: ./img/data_audit/data_audit_ex.png
 
 
-.. hint::
+History of Changes
+------------------
 
-    The set of entities, for which data audit is defined by the system administrator. Any entity can be 
-    :ref:`set as auditable <user-guide-entity-management-create-other>`.
+The link to the history of changes of a specific record is available in the top right corner of the record's 
+:ref:`View page <user-guide-ui-components-view-pages>`, if this is permitted to the user (the 
+:ref:`Data Audit capabilty <admin-capabilities-data-audit>` has been enabled for 
+at least one of the user's :ref:`roles <user-guide-user-management-permissions-basic>`)
```

Comment:
Comment: The phrase "if this is permitted to the user" is a bit awkward and unclear. What is being permitted exactly? The link to the record's history? Could this be clarified some?